services/spar/src/Spar/Intra/Brig.hs

```diff
@@ -49,6 +49,7 @@ module Spar.Intra.Brig
     createBrigUserNoSAML,
     updateEmail,
     getZUsrOwnedTeam,
+    authorizeScimTokenManagement,
```

Comment:
Why not `Spar.Intra.Galley.assertHasPermission`?

Reply:
`authorizeScimTokenManagement` uses `Spar.Intra.Galley.assertHasPermission`
Can't use `Spar.Intra.Galley.assertHasPermission` directly because the team of the user has to be fetched first.